docs/introduction.tex

```diff
@@ -150,7 +165,8 @@ \subsubsection{Modularity and Extensibility}
 behaviors are confined in the software.
 Also, typical \gls{NFC} simulators model
 fuel cycles (e.g. once-through, continuous reprocessing)
-with immutable connections between facilities. This enables \Cyclus to simulate any system
+with immutable connections between facilities. 
+This enables \Cyclus to simulate any system
```

Comment:
This sentence is still confusing. The paragraph makes it seem that Cyclus ALSO has immutable connections between facilities. This is not the case. Previous to the sentence beginning "This enables" there should be a sentence indicating that Cyclus is not one of the aforementioned simulators that immutably connects facilities.